CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing
+
+Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP or Mojang's obfuscation maps)
+is **completely forbidden** without explicit permission from the owners of said mappings. Discussing the naming approaches used in said projects
+is also not welcome - you have been warned. However, it is a good idea to consult name changes with other people - use pull requests or our community spaces to ask questions!
```

Comment:
Naming approach should be fine, but you cannot use their product as an evidence or a premise of your name

Reply:
That part is just straight from the README anyways, but I can take that out.

Reply:
That's good then.